src/content/protocols/spark.md

```diff
@@ -0,0 +1,414 @@
+---
+protocol: "Spark"
+website: "https://spark.fi"
+x: "https://x.com/sparkdotfi"
+github: ["https://github.com/sparkdotfi"]
+defillama_slug: ["spark"]
+chain: "Ethereum"
+stage: 0
+reasons: []
+risks: ["L", "M", "H", "H", "L"]
+author: ["mmilien_"]
+submission_date: "2025-09-04"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Spark is a star within the Sky ecosystem and consists of three main products:
+
+- Savings: A fixed savings rate on `USDS`.
+- SparkLend: a `USDS` and `DAI` centric lending protocol in which users can participate as lenders or borrowers. It combines liquidity from Sky and integrating with other DeFi protocols.
+- Spark Liquidity Layer: Direct liquidity provision into DeFi markets. This layer automates the liquidity provision of USDS, sUSDS, and USDC directly from Sky across various blockchain networks and DeFi protocols. It also enables Spark to provide liquidity into risk-adjusted yield strategies on other chains and protocols, as with Aave and Morpho on mainnet.
+
+# Overview
+
+## Chain
+
+SparkLend is deployed on Ethereum mainnet and the liquidity layer involves other various chains. This review focuses on the Ethereum mainnet deployment of all Spark products.
+
+> Chain score: Low
+
+## Upgradeability
+
+The upgradeability of the Spark protocol is limited. Most permissions are currently held by the Sky Governance.
+
+The Sky Governance could update the treasury (collected fees) or arbitrarily move funds out. As part of the Liquidity Management Layer (LML) the Sky Governance also may mint/burn `USDS` (debt) from the Sky protocol to Spark within set limits, which could lead to instability in the protocol. Two externally-owned accounts (EOA) also have the right to mint/burn `USDS`, bridge funds, and deposit them into Aave, Ethena, or vault-compatible protocols such as Morpho, but with stricter limits. Those EOAs are assumed to be corruptible with limited impact on the protocol because they are monitored and can be frozen by a _freezer_ multisig. Unfortunately, this multisig does not meet our security council requirements.
```

Comment:
can you focus on permissions with an impact for Spark users (not Sky users) of: loss of funds, loss of unclaimed yield, material change in expected protocol performance. 

optimally structure this as follows:

- paragraph 1: Impact of permissions over upgradeable contracts (loss of funds, loss of unclaimed yield, material change in expected protocol performance)
- paragraph 2: Impact of other permissions (loss of funds, loss of unclaimed yield, material change in expected protocol performance)

avoid language like: "unfortunately" etc

Reply:
also, in the diagram I cannot find EOAs with permission to mint/burn USDS